src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -249,17 +249,14 @@ public Map<UUID, Set<IamRole>> listAuthorizedSnapshots(AuthenticatedUserRequest
    * @param userReq authenticated user
    * @return accessible snapshot UUIDs and the IamRoles dictating their accessibility, established
    *     indirectly by any linked RAS passport
+   * @throws ParseException
```

Comment:
My usual javadoc rule of thumb is to omit anything that doesn't have a doc text associated with it. If we did ever generate javadoc for this code, it would have an empty `throws` generated automatically, so adding an empty form doesn't have any benefit.